SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -13,6 +13,16 @@ FSpatialTestSettings::FSpatialTestSettings()
 {
 }
 
+const FString FSpatialTestSettings::OverrideSettingsFileExtension = TEXT(".ini");
+const FString FSpatialTestSettings::OverrideSettingsFileDirectoryName = TEXT("MapSettingsOverrides");
+const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides");
```

Comment:
Could we have a separator here so the file names are more readable?
```suggestion
const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides_");
```